docs/source/roles/system_manager/manage_deployments.md

```diff
@@ -26,6 +26,16 @@ PS> ./Update_SRE_SSL_Certificate.ps1 -shmId <SHM ID> -sreId <SRE ID>
 - where `<SHM ID>` is the {ref}`management environment ID <roles_deployer_shm_id>` for this SHM
 - where `<SRE ID>` is the {ref}`secure research environment ID <roles_deployer_sre_id>` for this SRE
 
+(renew_sas)=
+
+## {{locked_with_key}} Renew SRE SAS tokens
+
+The [SRE storage containers](role_researcher_user_guide_shared_storage) for input data, backup and output are all provided by blob storage.
+The SRDs use [SAS tokens](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview) to authenticate and access the data.
+
+When the containers are deployed the SAS tokens are valid for one year.
```

Comment:
Is 1 year the right time? We had this as a generic "seems like long enough for most projects" placeholder. It could be that we want to cycle SAS tokens more regularly than that. Maybe not something to change in this PR though.